abi/known/tonstakers.json

```diff
@@ -0,0 +1,1245 @@
+        "op_code": "0xc9f04485",
+        "body": [
+          {
+            "name": "query_id",
+            "tlb_type": "## 64",
+            "format": "uint64"
+          },
+          {
+            "name": "interest_rate",
+            "tlb_type": "## 16",
+            "format": "uint16"
+          }
+        ]
+      },
+      {
+        "op_name": "tonstakers_operation_fee",
+        "op_code": "0x54d37487",
+        "body": [
+          {
+            "name": "query_id",
+            "tlb_type": "## 64",
+            "format": "uint64"
+          }
+        ]
+      },
+      {
+        "op_name": "tonstakers_request_notification",
+        "op_code": "0xb1ebae06",
+        "body": [
+          {
+            "name": "query_id",
+            "tlb_type": "## 64",
+            "format": "uint64"
+          },
+          {
+            "name": "min_loan",
+            "tlb_type": "Coins",
+            "format": "Coins"
+          },
+          {
+            "name": "max_loan",
+            "tlb_type": "Coins",
+            "format": "Coins"
+          },
+          {
+            "name": "max_interest",
+            "tlb_type": "## 24",
+            "format": "uint24"
+          }
+        ]
+      },
+      {
+        "op_name": "tonstakers_stats",
+        "op_code": "0xc1344900",
+        "body": [
+          {
+            "name": "query_id",
+            "tlb_type": "## 64",
+            "format": "uint64"
+          },
+          {
+            "name": "borrowed",
+            "tlb_type": "Coins",
+            "format": "Coins"
+          },
+          {
+            "name": "expected",
+            "tlb_type": "Coins",
+            "format": "Coins"
+          },
+          {
+            "name": "returned",
+            "tlb_type": "Coins",
+            "format": "Coins"
+          },
+          {
+            "name": "profit_sign",
+            "tlb_type": "## 1",
+            "format": "int1"
+          },
+          {
+            "name": "profit",
+            "tlb_type": "Coins",
+            "format": "Coins"
+          },
+          {
+            "name": "total_balance",
+            "tlb_type": "Coins",
+            "format": "Coins"
+          }
+        ]
+      }
+    ],
+    "get_methods": [
+      {
+        "name": "get_factory_addr",
+        "return_values": [
+          {
+            "name": "factory_addr",
+            "stack_type": "slice",
+            "format": "addr"
+          }
+        ]
+      },
+      {
+        "name": "get_owner_addr",
+        "return_values": [
+          {
+            "name": "owner_addr",
+            "stack_type": "slice",
+            "format": "addr"
+          }
+        ]
+      },
+      {
+        "name": "get_pool_addr",
+        "return_values": [
+          {
+            "name": "pool_addr",
+            "stack_type": "slice",
+            "format": "addr"
+          }
+        ]
+      },
+      {
+        "name": "get_pool_params",
```

Comment:
`get_pool_addr` and `get_pool_params` are copied from DeDust ABI and should not be here.